api/odl2.py

```diff
@@ -49,6 +53,21 @@ class ODL2Importer(OPDS2Importer, HasExternalIntegration):
 
     NAME = ODL2API.NAME
 
+    FEEDBOOKS_AUDIO = "{0}; protection={1}".format(
```

Comment:
This seems unrelated to the code here. Did this perhaps come in accidentally?

Reply:
Yes it seems they are using LICENSE_FORMAT directly from the ODLImporter class. I will remove both